db/migrations/000002_create-events-table.up.sql

```diff
@@ -0,0 +1,44 @@
+CREATE TABLE IF NOT EXISTS organizations (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(100) NOT NULL DEFAULT 'self'
```

Comment:
I think we should have a unique constraint in the organization name. Otherwise it's likely we'll have duplicates and would defeat the purpose of separating the table for normalization. 
Instead of "self" should it be the orgainization's name: "somos"?

Reply:
Yea, I thought I did add a unique constraint 🤔 .. but I will add it! Also it does make sense to have "SOMOS" instead of "self". Thanks!